components/raftstore/src/store/worker/read.rs

```diff
@@ -520,16 +524,26 @@ where
             // The local `ReadDelegate` is up to date
             Some(d) if !d.track_ver.any_new() => Some(Arc::clone(d)),
             _ => {
+                if self.get_must_track_version_changed_failpoint() {
+                    if let Some(d) = self.delegates.get(&region_id) {
+                        info!("must_track_version_changed: {}", d.track_ver.any_new());
```

Comment:
What does this failpoint suppose to do? Can we just add `debug` logs?

Reply:
This logic is not included, but only fix make clippy check.